src/components/NatureOfBusiness/NaicsResult.vue

```diff
@@ -0,0 +1,97 @@
+<template>
+  <v-row no-gutters class="naics-result" v-if="!!result" @click="emitClick()">
+    <v-col cols="2">
+      <div class="result-code">{{result.code}}</div>
+    </v-col>
+
+    <v-col cols="10">
+      <div class="d-flex justify-space-between align-top">
+        <label class="result-class-title">{{result.classTitle}}</label>
+        <v-btn large depressed color="primary" class="mt-n2">Select</v-btn>
+      </div>
+
+      <div class="result-class-definition">{{result.classDefinition}}</div>
+
+      <template v-if="elements.length > 0">
+        <div class="sample-activities">Sample activities:</div>
+        <ul>
+          <template v-for="(element, index) in elements">
+            <li v-if="showMore || (index < 5)" :key="index">
+              {{capitalize(element.elementDescription)}}
+            </li>
+          </template>
+        </ul>
+
+        <div v-if="!showMore && (elements.length >= 5)" class="show-more" @click.stop="showMore = true">
+          Show more...
+        </div>
+      </template>
+    </v-col>
+  </v-row>
+</template>
+
+<script lang="ts">
+import { Component, Emit, Prop, Vue } from 'vue-property-decorator'
+import { NaicsElementIF, NaicsResultIF } from '@bcrs-shared-components/interfaces'
```

Comment:
This _should_ work. We'll see.
(Same in NaicsResults.vue.)

Reply:
Oh wait... "@bcrs-shared-components" might point to the main node_modules, not my component's node_modules. But the unit tests and Storybook worked fine. Hmm...

Reply:
Not sure either, worth a try tho!